Clarify database health status (#2370)
neo4j/docs-operations: modules/ROOT/pages/clustering/monitoring/endpoints.adoc

| `votingMembers`                | string[] | no     | `[]`                                   | A member is considered a voting member when the leader has been receiving communication with it.

                                                                                              List of member's `memberId` that are considered part of the voting set by this primary.

> | `isHealthy`                      | boolean  | no     | `true`                                 | Indicates that the local database on this cluster member has not encountered a critical error that would prevent local writes.

Author: Yep, it's true. We don't actually use the word `panic` here. You'll find it only in the configs names. So, I can suggest other variants:

Indicates that the local database on this cluster member has not encountered a critical error that stops it from operating.

Indicates that the local database on this cluster member remains accessible/online/ and has not encountered a critical error.

Indicates that the local database on this cluster member has not encountered a critical error that blocks database operation.

> | `isHealthy`                      | boolean  | no     | `true`                                 | Indicates that the local database on this cluster member has not encountered a critical error that would prevent local writes.

Author: ```suggestion
| `isHealthy`                      | boolean  | no     | `true`                                 | Indicates that the local database on this cluster member has not encountered a critical error that can block database operation.
```

[CAUTION]
====
The `isHealthy` status applies only to this member's local database.

> It does not reflect the overall health of the cluster.

Reviewer: ```suggestion
It does not reflect the overall health of the database if it is clustered.
```
